Mathlib/Data/Bool/Basic.lean

```diff
@@ -82,6 +82,18 @@ theorem decide_eq {p q : Prop} [Decidable p] [Decidable q] : decide p = decide q
 theorem not_false' : ¬false := fun.
 #align bool.not_ff Bool.not_false'
 
+-- Porting note: new theorem
+theorem eq_iff_eq_true_iff {a b : Bool} : a = b ↔ ((a = true) ↔ (b = true)) :=
```

Comment:
Should the two sides of the iff be flipped?

Reply:
No, I think that the simp normal form for anything involving `Bool` should be to handle them like Props.